src/memory.md

```diff
@@ -4,17 +4,31 @@ description: How to import and export memory, and all the details on its layout.
 
 # Memory
 
-Similar to other languages that use linear memory, all data in AssemblyScript becomes stored at a specific memory offset so other parts of the program can read and modify it.
+Similar to other languages that use linear memory, and until the [GC](https://github.com/WebAssembly/gc) 🦄 proposal becomes available, all data in AssemblyScript is stored at a specific offset in linear memory so other parts of the program can read and modify it.
```

Comment:
I'm glad that you sprinkled these in the docs :smile: I think it helps to show it's more that we are trying to build on top of the standards and wasm building blocks, more than just being a young language (Both are true, but you know what I mean :joy: )